examples/SimplestExample.html

```diff
@@ -4,8 +4,8 @@
 <head lang="en">
     <meta charset="UTF-8">
     <title>WorldWind Example</title>
-    <!-- Include the Web WorldWind library. -->
-    <script src="http://worldwindserver.net/webworldwind/worldwind.min.js" type="text/javascript"></script>
+    <!-- Include the Web WorldWind library. You must first compile the library by running `npm run build` -->
+    <script src="../worldwind.min.js" type="text/javascript"></script>
```

Comment:
Nice catch. This hard-coded path was well-intentioned (demonstrate how to use the hosted library) it has been a problem. That said, I'm not sure that worldwind.min.js will be placed in the parent directory in our deployment scenario. Could we use the shim?

Also, we can drop the additional comment; @AkeluX and I have plans to ensure this step isn't needed for folks exploring WorldWind.

Reply:
I believe the shim would require use of RequireJS, right? That change would then require all of the RequireJS boilerplate. Implementation wise no big deal, but this is one a few examples which demonstrates the library without RequireJS.

I know it is a bit presumptuous, but what about the actual (future) hosted library? https://files.worldwind.arc.nasa.gov/web/0.9.0/worldwind.min.js

The [Standalone](https://github.com/NASAWorldWind/WebWorldWind/blob/develop/examples/Standalone.html#L24) example also uses a relative path to the minified version. Do we need to change anything there?

Reply:
Of course, the use of RequireJS is an oversight on my part. Let's deploy the examples in the same relative folder structure as appears in the development environment. In this case, worldwind.min.js is guaranteed to be in the parent folder.